app/navigators/live-activity-announcement/live-activity-announcement-stack.tsx

```diff
@@ -28,13 +24,19 @@ export const LiveAnnouncementNavigator = () => (
   <LiveAnnouncementStack.Navigator
     screenOptions={({ navigation }) => ({
       headerTransparent: true,
-      headerLeft: () => (
-        <CloseButton
-          onPress={() => navigation.navigate("planner")}
-          iconStyle={{ width: 32.5, height: 32.5, tintColor: "white", opacity: 0.5, marginTop: 8 }}
-        />
-      ),
-      headerBackground: () => <LiveAnnouncementHeaderBackground />,
+      // headerLeft: () => (
+      //   <CloseButton
+      //     onPress={() => navigation.navigate("planner")}
+      //     iconStyle={{
+      //       width: 32.5,
+      //       height: 32.5,
+      //       tintColor: Platform.select({ ios: "white", android: "grey" }),
+      //       opacity: 0.5,
+      //       marginTop: 8,
+      //     }}
+      //   />
+      // ),
```

Comment:
Why is it commented out?

Reply:
I did this to use the arrow icon for Android (close icon feels less suitable there). I'll fix it so it'll appear only for iOS. 
<img width="287" alt="Screenshot 2023-07-22 at 19 02 48" src="https://github.com/better-rail/app/assets/13344923/1d909a55-3c8f-4e18-adf9-cbf1d75116f0">